pkg/cmd/cli/describe/describer.go

```diff
@@ -345,6 +345,15 @@ func describeCustomStrategy(s *buildapi.CustomBuildStrategy, out *tabwriter.Writ
 }
 
 func describeJenkinsPipelineStrategy(s *buildapi.JenkinsPipelineBuildStrategy, out *tabwriter.Writer) {
+	if len(s.JenkinsfilePath) != 0 {
+		formatString(out, "Jenkinsfile path", s.JenkinsfilePath)
+	}
+	if len(s.Jenkinsfile) != 0 {
+		formatString(out, "Jenkinsfile contents", s.Jenkinsfile)
```

Comment:
it appears to print the full contents, though it does some newline related processing:

```
			fmt.Fprintf(out, "Dockerfile:\n")
			for _, s := range strings.Split(*p.Source.Dockerfile, "\n") {
				fmt.Fprintf(out, "  %s\n", s)
			}
```

I'll mimic this for the Jenkinsfile.